enterprise/dev/ci/internal/ci/pipeline.go

```diff
@@ -78,10 +78,11 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 	case c.isPR() && c.isGoOnly():
 		// If this is a go-only PR, run only the steps necessary to verify the go code.
 		pipelineOperations = []func(*bk.Pipeline){
-			addGoTests,            // ~1.5m
-			addCheck,              // ~1m
-			addGoBuild,            // ~0.5m
-			addPostgresBackcompat, // ~0.25m
+			addBackendIntegrationTests(c), // ~11m
```

Comment:
```suggestion
			addBackendIntegrationTests(c), // ~11m (optional via backend-dry-run and master-dry-run branch prefix)
```

Wondering if a comment is a good idea here, so it doesn't look like this is always run?